### ISSUE TITLE
Expand jets for tttt

### ISSUE THREAD

**Issue by bryates** [TopEFT/topcoffea#100]
We need add more jet categories to account for the `tttt` process.
e.g. `2lss` `8j` with `4b`

This is a lower priority, so let's address it after the next few PRs are merged.

**Comment by kmohrman** [comment]
For documentation purposes, here is an update on this issue. 

The idea described in this issue ended up evolving into the creation of a new SR designed to target 4t by requiring more b jets. The initial implementation for the 2lss regions was merged in @Dovahkinsl's PR #157. When testing the impact of the new SR on the sensitivity, we observed an improvement for the 4 4-heavy WCs as expected (no other WCs were significantly impacted). Specifically, for a test that included the full Run 2 signal samples (no data, no backgrounds, no systematics), the improvement was as follows (note that the fact that the improvement is less significant for the `lj0pt` fit is perhaps not surprising, since `lj0pt` already does a significantly better job for those WCs than njets does, so there is less room for improvement.:
* For `njets`: An improvement of ~15-25% for the 4-heavy WCs (no significant impacts on the others)
* For `lj0pt`: An improvement of ~10% for the 4-heavy WCs (no significant impacts on the others).


The next steps will be to implement similar categories for the 3l bins. This is in progress by @Dovahkinsl.
